main/res/drawable/ok.xml

```diff
@@ -0,0 +1,11 @@
+<!-- taken from material.io icons / check -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I'm quite sure we already have a checkmark icon inside cgeo... IIRC it's `ic_menu_done`

Reply:
ok, exchanged it for ic_menu_done, removed `ok.xml`